R/utils.R

```diff
@@ -76,4 +86,4 @@ rename_mapping <- function(mapping) {
 # Suggestion from https://stackoverflow.com/questions/9439256/how-can-i-handle-r-cmd-check-no-visible-binding-for-global-variable-notes-when
 # The reason to set globalVariables instead of define x, y, ... is because
```

Comment:
Interesting, nice little bit of optimization. Did you notice significant reduction in benchmarks?

Reply:
WoW. What happens here? The code is optimized?